solana/Cargo.toml

```diff
@@ -5,7 +5,11 @@ members = [
 ]
 
 [profile.release]
+opt-level = 3
 lto = true
+debug = false
```

Comment:
where did the changes in this file come from?

Reply:
Brett around 5 months ago... Think these are changes from the gateway_v2 branch